meilisearch/src/routes/indexes/search.rs

```diff
@@ -82,6 +82,64 @@ pub struct SearchQueryGet {
     matching_strategy: MatchingStrategy,
 }
 
+pub mod multiple_search {
```

Comment:
I don’t really see the point of doing this module instead of just dropping the function in the current module.

Also we usually try to follow the routes we’re in, thus, if the route associated to this module is `/search`, it shouldn’t be in `/indexes/search`.

I would prefer to;
1. Get rids of this module
2. Move the content of the module in `routes/mod.rs` or create a new file containing this code in `routes/multi_search.rs`

Reply:
The module is used to separate the multi query search from index query.

It reuses [`add_search_rules`](https://github.com/meilisearch/meilisearch/blob/ad49a8121ddb6ba97b57663fdb0bbf65252a3fb5/meilisearch/src/routes/indexes/search.rs#L176) from the parent module. Should I duplicate this function as well or move it to `meilisearch/src/search.rs`?

Reply:
I would say we move it to `/search.rs` since that already contains all the « helper » function we need to make both the get and post search 🤔

Reply:
done